dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/messaging/EmailMessageSender.kt

```diff
@@ -0,0 +1,46 @@
+package org.dataland.datalandbackend.services.messaging
+
```

Comment:
In der doc der Klasse steht dass das Ding für bulk und single zuständig ist.
Aber es gibt dann nur eine Funktion für single ?

Reply:
Selber Copy-Paste-Fehler

Reply:
Dann korrigiert bitte die docs

Reply:
Auch hier: bereits umgesetzt?

Reply:
![image](https://github.com/d-fine/Dataland/assets/99255296/db25d7c3-a2dd-4d5e-ad16-a18705722383)

Reply:
erledigt -> diesmal Wirklich :)